contracts/evm/contracts/upgradeable/UUPSProxy.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Can we use ERC1967Proxy directly? instead of having this empty implementation?

Reply:
We changed upgradaebility to Transparent upgrade so, this is removed!